src/app/questions/questions.component.ts

```diff
@@ -20,11 +18,15 @@ import { Subscription } from 'rxjs/Subscription';
   templateUrl: './questions.component.html',
   styleUrls: ['./questions.component.css']
 })
-export class QuestionsComponent implements OnInit {
+export class QuestionsComponent implements OnInit, OnDestroy {
 
   questions: Observable<Question[]>;
   question: Observable<Question>;
 
+  answersOfSelectedQuestion: Observable<Answer[]>;
+
+  subscriptions: Subscription[] = [];
+
```

Comment:
This seems to always be an array of one 😀  Please change this to `subscription: Subscription`